### DISCUSSION TITLE
[iOS][Safari][iCal] default calendar is not select

### DISCUSSION THREAD

**Discussion by DaveLomber** [add2cal/add-to-calendar-button#176] [Q&A]
Hi everyone, 

Recently we started use this lib in iOS Safari and face one UX issue

Whenever a user selects iCal option, it works well and opens a screen to add an event:

![IMG_9488](https://user-images.githubusercontent.com/39332586/189488851-46431e80-156b-499c-bec5-85e221e8cae2.PNG)

The issue here is that if a user will press Done, the event will be created, but will be hidden for CALENDAR APP.

To FIX this, it needs a user to go to 'Add To Calendar' screen, select a calendar and press Done, so then the Calendar section will appear:

![IMG_9489](https://user-images.githubusercontent.com/39332586/189488860-811347ca-ce86-4a0b-9f6b-dc5ab70e0adc.PNG)

Unfortunately this is non obvious for end user, so it leads to missed events in reality. 

Is there any way to improve the UX of this flow? 

Thanks in advance

**Comment by jekuer** [comment]
Totally agree.
However, this is the default behavior of the Apple Calendar. As far as I know, we cannot change that behavior - but maybe somebody else got an idea?

A UX workaround could be to add an additional step before triggering the download, where the user gets informed about where to click on the following screen? Could help, but 1 more click and screen.

One question though: Do you have the default Calendar set in the settings?

**Reply by noblessem** [reply]
Yes, and it doesn`t help at all(

**Reply by noblessem** [reply]
Default Calendar setting working just when u creating event manually

**Reply by DaveLomber** [reply]
yes I have the default calendar, but this doesn't help

**Reply by jekuer** [reply]
Investigated this a little bit. I do not see any chance at the moment to make this work, since this seems to be the default behavior of Apple, when dealing with iCal files.
Please let me know, whenever you see a different behavior with other similar tools! This would help a lot.

**Reply by DaveLomber** [reply]
So far we did not find a way how to manage it in a better way. Instead, we show some hints to end users:

![IMG_2E94571C7176-1](https://user-images.githubusercontent.com/39332586/193525335-c8b0c3c9-f754-43c0-8313-31e5197dd412.jpeg)
